test/sqllogictest/select.slt

```diff
@@ -0,0 +1,25 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+mode cockroach
+
+statement ok
+CREATE TABLE foo (a integer)
+
+statement ok
+INSERT INTO foo VALUES (42)
+
+query I
+SELECT * FROM foo
+----
+42
+
+# over qualified objects should return an error
+query error qualified name did not have between 1 and 3 components
+SELECT * FROM universe.db.schema.foo
```

Comment:
This isn't really testing `SELECT` in a meaningful way; this is looking at a function of name resolution. I'd recommending finding a way to wedge this test in a framework more akin to that.

Reply:
I moved this to a new `slt` called `name_resolution.slt`, which is probably more appropriate. Or did you have a different type of test in mind?